src/Assets.php

```diff
@@ -81,12 +81,15 @@ public static function register_assets() {
 		self::register_script( 'wc-price-filter', plugins_url( self::get_block_asset_build_path( 'price-filter' ), __DIR__ ), $block_dependencies );
 		self::register_script( 'wc-attribute-filter', plugins_url( self::get_block_asset_build_path( 'attribute-filter' ), __DIR__ ), $block_dependencies );
 		self::register_script( 'wc-active-filters', plugins_url( self::get_block_asset_build_path( 'active-filters' ), __DIR__ ), $block_dependencies );
-		self::register_script( 'wc-checkout-block', plugins_url( self::get_block_asset_build_path( 'checkout' ), __DIR__ ), $block_dependencies );
-		self::register_script( 'wc-cart-block', plugins_url( self::get_block_asset_build_path( 'cart' ), __DIR__ ), $block_dependencies );
 
-		if ( 'experimental' === WOOCOMMERCE_BLOCKS_PHASE ) {
+		if ( WOOCOMMERCE_BLOCKS_PHASE > 2 ) {
 			self::register_script( 'wc-single-product', plugins_url( self::get_block_asset_build_path( 'single-product' ), __DIR__ ), $block_dependencies );
 		}
+
+		if ( WOOCOMMERCE_BLOCKS_PHASE > 1 ) {
+			self::register_script( 'wc-checkout-block', plugins_url( self::get_block_asset_build_path( 'checkout' ), __DIR__ ), $block_dependencies );
+			self::register_script( 'wc-cart-block', plugins_url( self::get_block_asset_build_path( 'cart' ), __DIR__ ), $block_dependencies );
+		}
```

Comment:
At some point we should move script registration to the block classes - I always forget to add them here :)

Reply:
Agreed, there's a bunch of things I'd like to do around that eventually (including incorporating using `block.json` files which can contain all the things around the blocks and thus make the initialization code more dynamic).